### PR TITLE
Added methods constants

### PR DESCRIPTION
**Pull request by toby3d**
It can little bit reduce compile time and help avoid typos.

### CODE REVIEW

File: methods.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package fasthttp
+
+// HTTP methods were stolen from net/http.
```
Comment (erikdubbelboer) [inline methods.go]: Please change to `copied from` :)

Comment (toby3d) [reply methods.go]: Fixed :ok_hand:

**Comment by erikdubbelboer** [comment]
Thanks

**Comment by kirillDanshin** [comment]
We need to add Go license, e.g. in `/third-party_licenses/golang`, and apply this license to the new file

**Comment by erikdubbelboer** [comment]
Or we just remove that one line. This is such basic code and doesn't affect any functionality. I don't think anyone will care.

**Pull request merged**